src/client.c

```diff
@@ -417,7 +418,7 @@ int trilogy_auth_switch_send(trilogy_conn_t *conn, const trilogy_handshake_t *ha
 void trilogy_auth_clear_password(trilogy_conn_t *conn)
 {
     if (conn->socket->opts.password) {
-        memset(conn->socket->opts.password, 0, conn->socket->opts.password_len);
+        memset_s(conn->socket->opts.password, conn->socket->opts.password_len, 0, conn->socket->opts.password_len);
```

Comment:
This function is not always defined, so it would need a detection mechanism to see if it's available.

Reply:
yup realized too late that `memset_s` is not in posix. I believe `explicit_bzero` would be the alternative. Need to do an if def to handle this 😑